### PR TITLE
list observationPeriods in summary_report.html

### LINKED ISSUE #103: StatVar series are not grouped by observationPeriods
e.g., the series has P3M, but that is not to be seen in https://storage.cloud.google.com/unresolved_mcf/fbi/hate_crime/table14/summary_report.html ... it is in https://storage.cloud.google.com/unresolved_mcf/fbi/hate_crime/table14/report.json

### PR DESCRIPTION
**Pull request by enjoythecode**
Fixes #103 

- added an observationPeriods column in the "StatVarObservations by
  StatVar" and "Series Summaries for Sample Places" tables in
  SummaryReport.ftl
- observationPeriod values are added in StatChecker.java and
  PlaceSeriesSummary.java
- observationPeriods value is pulled from new getObservationPeriods()
method in SummaryReportGenerator.java
- re-generated test goldens for summary_report.html

### CODE REVIEW

File: tool/src/test/resources/org/datacommons/tool/genmcf/statchecks/output/summary_report.html [hunk 1]
```diff
@@ -154,6 +155,9 @@ <h2>StatVarObservations by StatVar</h2>
               <td>
                 <div></div>
               </td>
+              <td>
+                <div></div>
```
Comment (pradh) [inline tool/src/test/resources/org/datacommons/tool/genmcf/statchecks/output/summary_report.html:159]: Can you modify an input node to include observationPeriod so we can see it in the output?

**Review by pradh — COMMENTED** [review]
Thanks for the change!

File: tool/src/test/resources/org/datacommons/tool/genmcf/statchecks/output/summary_report.html (hunk 1 above)
Comment (enjoythecode) [reply tool/src/test/resources/org/datacommons/tool/genmcf/statchecks/output/summary_report.html]: thank you for the feedback, please see:
- this summary_report [(raw html)](https://github.com/enjoythecode/import/blob/01cc744aa83b1fa4d1a3cb061bd6f7b0e1cb63a2/tool/src/test/resources/org/datacommons/tool/genmcf/statchecks/output/summary_report.html) [(rendered html)](https://htmlpreview.github.io/?https://github.com/enjoythecode/import/blob/01cc744aa83b1fa4d1a3cb061bd6f7b0e1cb63a2/tool/src/test/resources/org/datacommons/tool/genmcf/statchecks/output/summary_report.html) at `geoId/07`

File: tool/src/test/resources/org/datacommons/tool/genmcf/statchecks/input/covid.csv [hunk 2]
```diff
@@ -1,50 +1,50 @@
...
-2020-06-02,geoId/211,1
-2020-06-02,geoId/221,1
-2020-06-02,geoId/231,1
-2020-06-02,geoId/241,1
-2020-06-02,geoId/25111,1
-2020-06-02,geoId/26111,1
-2020-06-02,geoId/27111,1
-2020-06-02,geoId/28111,1
\ No newline at end of file
+DateTime,geoId,CumulativeCount_MedicalTest_ConditionCOVID_19_Positive, Observation_Period
```
Comment (pradh) [inline tool/src/test/resources/org/datacommons/tool/genmcf/statchecks/input/covid.csv:1]: nit: remove space after comma?

File: tool/src/test/resources/org/datacommons/tool/genmcf/statchecks/output/report.json [hunk 3]
```diff
@@ -122,94 +114,6 @@
         }]
       }]
     }]
-  }, {
```
Comment (pradh) [inline tool/src/test/resources/org/datacommons/tool/genmcf/statchecks/output/report.json:125]: The current changes affect these statcheck counters that were designed to validate these counters.  Could you please use a single observationPeriod for all existing rows?  If you want to multiple periods, add new rows.

Comment (enjoythecode) [reply tool/src/test/resources/org/datacommons/tool/genmcf/statchecks/output/report.json:125]: fixed, please see:
- this summary_report [(raw html)](https://github.com/enjoythecode/import/blob/fix103/tool/src/test/resources/org/datacommons/tool/genmcf/statchecks/output/summary_report.html) [(rendered html)](https://htmlpreview.github.io/?https://github.com/enjoythecode/import/blob/fix103/tool/src/test/resources/org/datacommons/tool/genmcf/statchecks/output/summary_report.html) at `geoId/01`

**Review by pradh — APPROVED** [review]
Thanks for the fix!

**Comment by pradh** [comment]
Could you please resolve conflicts so its mergeable?

**Comment by enjoythecode** [comment]
I was just merging as you commented. Conflicts are now resolved +1

**Pull request merged**